### PR TITLE
Fixes #10681: Complete documentation introduction

### PR DESCRIPTION
**Pull request by fanf**
https://www.rudder-project.org/redmine/issues/10681

### CODE REVIEW

File: 00_introduction/20_key_features.txt [hunk 1]
```diff
@@ -3,48 +3,144 @@
 
 ==== Os independent target configuration state definition
```
Comment (amousset) [inline 00_introduction/20_key_features.txt]: OS

File: 00_introduction/20_key_features.txt [hunk 2]
```diff
@@ -3,48 +3,144 @@
...
 
-image::./images/core_techniques.png[Standard Technique Library]
-
-
-
-==== Centralize and aggregate real configuration state
+Rudder is able to adapt to complex process and only do the minimal required
+work so that the server converge to the desired state, and so whatever was the
```
Comment (amousset) [inline 00_introduction/20_key_features.txt]: converges

File: 00_introduction/20_key_features.txt [hunk 3]
```diff
@@ -3,48 +3,144 @@
...
+starting state point. Rudder works as a GPS would, adapting the path to your
+destination depending of the path you actually took. This process is much more
+resilient to changes than a step by step, procedural description of the command
```
Comment (amousset) [inline 00_introduction/20_key_features.txt]: commands

File: 00_introduction/20_key_features.txt [hunk 4]
```diff
@@ -3,48 +3,144 @@
...
+to execute.
+
+Rudder is nativelly integrated with the supported OS (Linux, Windows, AIX - see
```
Comment (amousset) [inline 00_introduction/20_key_features.txt]: natively

File: 00_introduction/20_key_features.txt [hunk 5]
```diff
@@ -3,48 +3,144 @@
...
+[TODO]) so that it provides generic, abstract, OS independant primitives to the
+user who can:
+
+image::./images/core_techniques.png["Standard Technique Library", float="right"]
+
+* install software in OS native packaging system (RPM on RHEL, Windows software
+  components, or even direct install from sources),
+* configure OS level parameters and services like logs, DNS, NTP, etc
```
Comment (amousset) [inline 00_introduction/20_key_features.txt]: etc.

File: 00_introduction/20_key_features.txt [hunk 6]
```diff
@@ -3,48 +3,144 @@
...
+* build an hardened system by configuring and then continuously verifying the
+  correct set-up of security rules like file system rights, file integrity
+  checking, etc.
+* configure middleware by files (for example in Linux world, whatever the file
+  format, and be it from a template or by only specifying enforcement of some
+  configuration parameters) or thanks to the Windows Regestry,
+* manage service start-up at boot time and ensure that a service is correctly
+  running at any time, starting it up again if needed.
+
+The simple primitives can be simply mixed and todo_link[extended] to provided
```
Comment (amousset) [inline 00_introduction/20_key_features.txt]: provide

File: 00_introduction/20_key_features.txt [hunk 7]
```diff
@@ -3,48 +3,144 @@
...
+solutions for any and all of your unique use cases of software stacks,
+deployments, IT services or configuration that can't be nativelly supported.
```
Comment (amousset) [inline 00_introduction/20_key_features.txt]: natively

File: 00_introduction/20_key_features.txt [hunk 8]
```diff
@@ -3,48 +3,144 @@
...
 
 image::./images/introduction/general_behavior_workflow.png[Define target, check, report, remediate]
 
+Rudder can also *continuously check* that rules are valid and *proactively* correct
+any drift from the desired application state when needed. A *graphical reporting*
+displays what happened and when.
+
 image::./images/introduction/rules_compliance.png[Rules compliance reporting]
 
+Rudder can notify the ops team about a drift from the desire configuration state.
```
Comment (amousset) [inline 00_introduction/20_key_features.txt]: desired

File: 00_introduction/20_key_features.txt [hunk 9]
```diff
@@ -3,48 +3,144 @@
...
+Understanding what the problem is is made simpler by the graphical reporting
+which allows to drill down toward the technical root cause and see in a blink
+where the drift come from.
```
Comment (amousset) [inline 00_introduction/20_key_features.txt]: comes

File: 00_introduction/20_key_features.txt [hunk 10]
```diff
@@ -3,48 +3,144 @@
...
+
 image::./images/introduction/rule_compliance_details.png[Fine grained reporting on configuration components]
 
 
 ==== Automatic inventory
 
+Rudder automatically does a technical, detailled inventory of the servers on
```
Comment (amousset) [inline 00_introduction/20_key_features.txt]: detailed

File: 00_introduction/20_key_features.txt [hunk 11]
```diff
@@ -3,48 +3,144 @@
...
+All Rudder commands are available throught an exhaustive REST API. That API is
+http://www.rudder-project.org/rudder-api-doc/[fully documented online] and can
+be used to todo_link[quickly and smoothly integrate Rudder with you existing infrastructure].
 
 
 ==== Audit trace and Change Requests
 
+Any change done thanks to Rudder in your infrastructure is automatically
+recorded in an *Audit Log* which allows a full traceability of all changes.
+That feature also allow rollbacks of the recorded change.
```
Comment (amousset) [inline 00_introduction/20_key_features.txt]: allows

File: 00_introduction/20_key_features.txt [hunk 12]
```diff
@@ -3,48 +3,144 @@
...
+
 image::./images/introduction/audit_trace.png[Trace events and display changes]
 
+All change can be forced to go through a peer review or validation step and
```
Comment (amousset) [inline 00_introduction/20_key_features.txt]: changes

File: 00_introduction/20_key_features.txt [hunk 13]
```diff
@@ -3,48 +3,144 @@
...
+OS independent, parametrizable configuration modules. By combining these module,
+you are able to manage any configuration and build advanced configuration
+policies for your IT services:
 
-image::./images/introduction/ncf_language.png[high level definition language]
+image::./images/introduction/rule_directive_generic_method_stack.png[Build your own configuration, matching your requirements]
 
-image::./images/introduction/technique_editor_overview.png[Graphical Technique Editor - the simplest way to build new configuration]
+The unitary configuration module can be configured thanks to a high level
+programation language:
```
Comment (amousset) [inline 00_introduction/20_key_features.txt]: programming

**Comment by amousset** [comment]
This PR also needs to be rebased.

**Comment by fanf** [comment]
PR rebased

**Comment by fanf** [comment]
PR rebased

File: 00_introduction/20_key_features.txt [hunk 14]
```diff
@@ -1,49 +1,146 @@
...
+programming language:
 
+image::./images/introduction/ncf_language.png[high level definition language]
 
+But the natural, common strategy to use them is with the provided graphical
+editor which allows to use all the same modules, but with a web UI and
+drag'n'drop. Of course, you can configure each unitary module to use data from
+a node and behave specifically on each one.
 
+image::./images/introduction/technique_editor_overview.png[Graphical Technique Editor - the simplest way to build new configuration]
```
Comment (amousset) [inline 00_introduction/20_key_features.txt]: Missing newline, it is what breaks the tests.

**Comment by fanf** [comment]
PR rebased

**Pull request merged**